Documentation/UserRoundTrip.rst

```diff
@@ -0,0 +1,345 @@
+documentation (see `News extension <https://docs.typo3.org/p/georgringer/news/main/en-us/>`__).
+
+For a Composer package, replace the TER URL with the Packagist URL. For a
+standalone manual, replace the TER URL with the documentation URL.
+
+The Packagist URL is linked indirectly from the TER page and is therefore not
+specified in this configuration file. The issues URL is considered here because
+it is closely related to the VCS repository aspect.
+
+
+.. _user-round-trip-vcs-repository-aspect:
+
+VCS repository
+--------------
+
+Guide the user from the VCS repository page to the other aspects by offering a
+table in the :ref:`README file <readme-rst>`:
+
+.. code-block:: md
+
+   |                  | URL                                            |
+   |------------------|------------------------------------------------|
+   | **Repository:**  | <vcs-repository>                               |
+   | **Read online:** | <documentation>                                |
+   | **TER:**         | <typo3-extension-repository>                   |
+
+for example for the *Mask* extension:
+
+.. code-block:: md
+
+   |                  | URL                                            |
+   |------------------|------------------------------------------------|
+   | **Repository:**  | https://github.com/Gernott/mask                |
+   | **Read online:** | https://docs.typo3.org/p/mask/mask/main/en-us/ |
+   | **TER:**         | https://extensions.typo3.org/extension/mask    |
+
+and find the README file usually displayed by the VCS host below the file list
+of the repository (see `Mask example <https://github.com/Gernott/mask>`__).
+
+For a Composer package, replace the TER URL with the Packagist URL and label it
+"Packagist:". For a standalone manual, remove the TER entry.
+
+The Packagist URL is linked indirectly from the TER page and is therefore not
+included in this configuration file.
+
+
+.. _user-round-trip-packagist-aspect:
+
+Packagist
+---------
+
+Guide the user from the Packagist page to the other aspects by setting these
+:file:`composer.json` configuration values:
+
+.. code-block:: none
+
+    "homepage": "<typo3-extension-repository>",
+    "support": {
+        "docs": "<documentation>",
+        "issues": "<vcs-repository-issues>",
+        "source": "<vcs-repository>"
+    }
+
+for example for the *Address List* extension:
+
+.. code-block:: none
+
+   "homepage": "https://extensions.typo3.org/extension/tt_address",
+   "support": {
+      "docs": "https://docs.typo3.org/p/friendsoftypo3/tt-address/main/en-us/",
+      "issues": "https://github.com/FriendsOfTYPO3/tt_address/issues",
+      "source": "https://github.com/FriendsOfTYPO3/tt_address"
+   }
+
+and find the links to the other aspects shown in the right column of the
+Packagist page (see `Address List example <https://packagist.org/packages/friendsoftypo3/tt-address>`__).
+
+For a Composer package, replace the TER URL with the Packagist URL.
```

Comment:
Is there such a thing as a Composer package / TER package? we usually recommend to offer both composer and TER support...?

Reply:
Composer package means something like [TYPO3 Console](https://packagist.org/packages/helhum/typo3-console), [Tailor](https://packagist.org/packages/typo3/tailor), [Surf](https://packagist.org/packages/typo3/surf), [Rector for TYPO3](https://packagist.org/packages/ssch/typo3-rector), etc. They don't have a mandatory ext_emconf.php.

Reply:
Yes, this is about TER or not TER here. Sticking to current state.